client/src/components/Sidebar.tsx

```diff
@@ -0,0 +1,85 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import { jsx, Flex } from "theme-ui";
+import { State } from "../reducers";
+import store from "../store";
+import { connect } from "react-redux";
+import { useState } from "react";
+import { useEffect } from "react";
+import { RestaurantsState } from "../reducers/restaurants";
+import { restaurantsFetch } from "../actions/restaurants";
+import { mapDataState } from "../reducers/mapdata";
+
+interface SidebarProps {
+    readonly restaurants: RestaurantsState
+    readonly mapData: mapDataState
+}
+
+function Sidebar({ restaurants, mapData }: SidebarProps) {
+    const restaurantData = "resource" in restaurants ? restaurants.resource : null;
+    const pag_count = Math.ceil((Number(restaurantData?.restaurants.count) / 5));
+    const [currentPag, setCurentPag] = useState<number>(1);
+    const prev = restaurantData?.restaurants.previous
+    const next = restaurantData?.restaurants.next
+    const allRestaurants = restaurantData?.restaurants.results;
+
+    useEffect(() => {
+        mapData.data.bounds && currentPag && store.dispatch(restaurantsFetch([mapData.data.bounds, currentPag]));
+        // eslint-disable-next-line
```

Comment:
It's in general better to be specific about the lint warning you're suppressing:
```suggestion
        // eslint-disable-next-line react-hooks/exhaustive-deps
```